packages/wdio-browserstack-service/src/cleanup.ts

```diff
@@ -0,0 +1,27 @@
+import { stopBuildUpstream } from './util'
+
+export default class BStackCleanup {
+    static startCleanup() {
+        try {
+            this.executeObservabilityCleanup()
```

Comment:
I don't see how `stopBuildUpstream` can ever throw an error since it has a `catch` callback attached to the promise.

Reply:
Removed the unnecessary try block